dubbo-compatible/src/test/java/org/apache/dubbo/generic/GenericServiceTest.java

```diff
@@ -108,7 +107,7 @@ public void testGenericComplexCompute4FullServiceMetadata() {
 
         FullServiceDefinition fullServiceDefinition = ServiceDefinitionBuilder.buildFullDefinition(DemoService.class);
         MethodDefinition methodDefinition = getMethod("complexCompute", fullServiceDefinition.getMethods());
-        Map parm2= createComplextObject(fullServiceDefinition,var1, var2, l, var3, var4, testEnum);
+        Map parm2= createComplexObject(fullServiceDefinition,var1, var2, l, var3, var4, testEnum);
```

Comment:
The name `parm2` looks weird to me. Could you improve it? And there should be a space before the equal sign.